python_docs_theme/static/pydoctheme.css

```diff
@@ -17,7 +26,7 @@ div.related a:hover {
     color: #0095C4;
 }
 
-div.related:first-child {
+div.related:first-of-type {
```

Comment:
I have not seen any occurence where you shifted the `div.related` or is this due to some jinja block before which you added stuff?

Reply:
Yes, I've added mobile menu before div.related, because I had to add it to the top of the `body` block.